api/test/pact/publish/publish.ts

```diff
@@ -5,44 +5,44 @@ import { getConfigValue } from '../../../configuration'
 import { PACT_BRANCH_NAME, PACT_BROKER_PASSWORD, PACT_BROKER_URL, PACT_BROKER_USERNAME, PACT_CONSUMER_VERSION } from '../../../configuration/references'
 
 const publish = async (): Promise<void> => {
-    try {
+  try {
 
     const pactBroker = getConfigValue(PACT_BROKER_URL) ?
       getConfigValue(PACT_BROKER_URL) : 'http://localhost:80'
 
-        const pactTag = getConfigValue(PACT_BRANCH_NAME) ?
-            getConfigValue(PACT_BRANCH_NAME) : 'Dev'
-
-        const consumerVersion = getConfigValue(PACT_CONSUMER_VERSION) !== '' ?
-            // @ts-ignore
-            getConfigValue(PACT_CONSUMER_VERSION) : git.short()
-
-        const certPath = path.resolve(__dirname, "../cer/ca-bundle.crt")
-        process.env.SSL_CERT_FILE = certPath
-
-        const opts = {
-            consumerVersion,
-            pactBroker,
-            pactBrokerPassword: getConfigValue(PACT_BROKER_PASSWORD),
-            pactBrokerUsername: getConfigValue(PACT_BROKER_USERNAME),
-            pactFilesOrDirs: [
-                path.resolve(__dirname, '../pacts/'),
-            ],
-            tags: [pactTag],
-        }
+    const pactTag = getConfigValue(PACT_BRANCH_NAME) ?
+      getConfigValue(PACT_BRANCH_NAME) : 'Dev'
+
+    const consumerVersion = getConfigValue(PACT_CONSUMER_VERSION) !== '' ?
+      // @ts-ignore
+      getConfigValue(PACT_CONSUMER_VERSION) : git.short()
+
+    const certPath = path.resolve(__dirname, "../cer/ca-bundle.crt")
+    process.env.SSL_CERT_FILE = certPath
+
+    const opts = {
+      consumerVersion,
+      pactBroker,
+      pactBrokerPassword: getConfigValue(PACT_BROKER_PASSWORD),
+      pactBrokerUsername: getConfigValue(PACT_BROKER_USERNAME),
+      pactFilesOrDirs: [
+        path.resolve(__dirname, '../pacts/'),
+      ],
+      tags: [pactTag],
+    }
 
-        await pact.publishPacts(opts)
+    await pact.publishPacts(opts)
 
-        console.log('Pact contract publishing complete!')
-        console.log('')
-        console.log(`Head over to ${pactBroker}`)
-        console.log('to see your published contracts.')
+    console.log('Pact contract publishing complete!')
+    console.log('')
+    console.log(`Head over to ${pactBroker}`)
+    console.log('to see your published contracts.')
```

Comment:
could use a multiline comment here or a console.group()

Reply:
Not my code. This was entirely to fix lint errors. All the PACT tests have been written by people outside of the ExUI team from what I can tell. I'll have a look, though.

Reply:
Sorted.